updater/README.md

```diff
@@ -18,6 +18,17 @@ The updater may be run in two modes:
 1. Create a config file in `updater/config.py` in your local clone (based on [`updater/config.py.example`](config.py.example))
 1. Configure a service or cronjob that executes `python3 update-stats.py` once per day.
 
+Alternatively you can set it up via Docker:
+1. Build the Docker image
+    ```sh
+    $ docker build . --tag hubble-updater
+    ```
+1. Create a config file in `updater/config.py` (based on [`updater/config.py.example`](config.py.example))
+1. Configure a service or cronjob that executes the docker container once a day (replace `/path/to/config.py` and `/path/to/key` with your paths):
```

Comment:
```suggestion
1. Configure a service or cron job that executes the Docker container once a day (replace `/path/to/config.py` and `/path/to/key` with your paths):
```

Reply:
Also, I’d recommend a `systemd` service over a cron job.

Reply:
Hmm. Unfortunately, Alpine doesn't use systemd ... therefore I think it is easier to just use cron. OK?

Reply:
Absolutely, didn’t know about that :+1:.